Gemfile

```diff
@@ -1,20 +1,20 @@
 source 'https://rubygems.org'
 
-gem 'rails', '4.2.7.1'
```

Comment:
Can you leave the version number in here; it's an easy shorthand to work out which version of rails we think this app is running on.  Intentionally or not by removing it you've bumped the app to rails 4.2.8, which isn't a problem but you didn't mention it in the commit note so perhaps you didn't intend it.